verification/curator-service/ui/src/components/App.tsx

```diff
@@ -531,12 +526,12 @@ export default function App(): JSX.Element {
                 >
                     <div className={classes.drawerHeader} />
                     <Switch>
-                        {user.email && (
+                        {user && (
                             <Route exact path="/cases">
                                 <LinelistTable user={user} />
                             </Route>
                         )}
-                        {hasAnyRole(['curator']) && (
+                        {user && hasAnyRole(['curator']) && (
```

Comment:
Can the user check be inside of hasAnyRole so this doesn't need to be added everywhere?

Reply:
no this is a typecheck for react, there is already a check in hasAnyRole for !user but apparently it is not enough.

Reply:
Gotcha. May not need it on this line anymore since we don't pass user into the SourceTable

Reply:
indeed, thanks. removed.